utils/index.js

```diff
@@ -7,6 +7,9 @@ const renderInlineImage = (file) => {
   return <div dangerouslySetInnerHTML={{ __html: `<img alt='${file.alt}' src='${file.url}' class='mb-4 border divide-gray-600' />` }} />
 }
 
+const randomQuote = quotes => quotes[Math.floor(Math.random() * (quotes.length - 1))]
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/dominion79/Saw-Front-End-App/pullRequest?prid=8928846)
```suggestion
const randomQuote = (quotes) => quotes[Math.floor(Math.random() * (quotes.length - 1))]
```